### PR TITLE
DOCS-4711 Private Locations Configuration Doc Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Consolidate information and remove static tables for parameter descriptions in the Help command.

### Motivation

DOCS-4711

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/private_locations/_index.md [hunk 1]
```diff
@@ -188,6 +176,10 @@ docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetic
 
 This command starts a Docker container and makes your private location ready to run tests. **Datadog recommends running the container in detached mode with proper restart policy.**
 
+<div class="alert alert-info">You can upload custom root certificates to your private locations to have your API and browser tests perform SSL handshake using your own `.pem` files with the <a href="https://hub.docker.com/r/datadog/synthetics-private-location-worker">Synthetics Private Location Worker</a> version 1.5.3 or later.</div>
```
Comment (HugoPct) [inline content/en/synthetics/private_locations/_index.md:179]: why is it in a "info" ?

File: content/en/synthetics/private_locations/_index.md [hunk 2]
```diff
@@ -212,6 +204,10 @@ This command starts a Docker container and makes your private location ready to
     docker-compose -f docker-compose.yml up
     ```
 
+<div class="alert alert-info">You can upload custom root certificates to your private locations to have your API and browser tests perform SSL handshake using your own `.pem` files with the <a href="https://hub.docker.com/r/datadog/synthetics-private-location-worker">Synthetics Private Location Worker</a> version 1.5.3 or later.</div>
```
Comment (HugoPct) [inline content/en/synthetics/private_locations/_index.md:207]: is it expected to have it 2 times?

Comment (alai97) [reply content/en/synthetics/private_locations/_index.md:207]: Yes, if you wanted the same message in two tabs. Do you want it to appear in one tab, and if so, which one?

File: content/en/synthetics/private_locations/_index.md (hunk 1 above)
Comment (alai97) [reply content/en/synthetics/private_locations/_index.md:179]: Want to do red (danger) or yellow (warning) instead?

Comment (drichards-87) [inline content/en/synthetics/private_locations/_index.md:179]:
```suggestion
<div class="alert alert-info">You can upload custom root certificates to your private locations to have your API and browser tests perform the SSL handshake using your own `.pem` files with the <a href="https://hub.docker.com/r/datadog/synthetics-private-location-worker">Synthetics Private Location Worker</a> version 1.5.3 or later.</div>
```

File: content/en/synthetics/private_locations/configuration.md [hunk 3]
```diff
@@ -13,165 +13,24 @@ further_reading:
...
-**Default**: `none`<br>
-Display full worker configuration parameters.
-
-`help`
-: **Type**: Boolean <br>
-**Default**: `none`<br>
-Show help.
-
-**Note**: Private Location containers output logs to stdout/stderr without saving them within the container.
+Arguments set in the launch command have precedence over the configuration file. However, these options are not stored and are consequently only prevalent for a given launch.
```
Comment (drichards-87) [inline content/en/synthetics/private_locations/configuration.md:30]: I know you didn't change the above text in this PR, but should "prevalent" be "relevant"?

**Review by drichards-87 — APPROVED** [review]
@alai97 Left you one very minor suggestion + a question. PR is approved.

File: content/en/synthetics/private_locations/configuration.md (hunk 3 above)
Comment (alai97) [reply content/en/synthetics/private_locations/configuration.md:30]:
```suggestion
Arguments set in the launch command have precedence over the configuration file. However, these options are not stored and are consequently only revelant for a given launch.
```

**Review by alai97 — COMMENTED** [review]
@HugoPct to supply primary parameters that we want to keep on this page.

**Review by HugoPct — COMMENTED** [review]
1 suggestion otherwise good to go

**Pull request merged**